inc/plugins/class-dashboard.php

```diff
@@ -266,6 +271,316 @@ private function the_otter_banner() {
 		<?php
 	}
 
+	/**
+	 * Hook the form submissions widget.
+	 *
+	 * @return void
+	 */
+	public function form_submissions_widget() {
+		wp_add_dashboard_widget(
+			'otter_form_submissions_widget',
+			__( 'Otter Blocks - Form Submissions', 'otter-blocks' ),
+			array( $this, 'form_submissions_widget_content' )
+		);
+	}
+
+	/**
+	 * Display the form submissions widget content.
+	 *
+	 * @return void
+	 */
+	public function form_submissions_widget_content() {
+
+		$is_active = Pro::is_pro_active();
```

Comment:
Thank you Rodica for noticing it, we can just check if the Pro is active or not if it's false then simply return the free code, no point in doing running all the logic that only needs in Pro to do unless we have Pro.